includes/Modules/Analytics.php

```diff
@@ -962,6 +962,29 @@ function ( $dimension_def ) {
 					}
 				}
 
+				$dimension_filters          = $data['dimensionFilters'];
+				$dimension_filter_instances = array();
+				if ( ! empty( $dimension_filters ) && is_array( $dimension_filters ) ) {
+					foreach ( $request_args['dimensions'] as $dimension ) {
+						$dimension_name = $dimension->getName();
+						if ( ! isset( $dimension_filters[ $dimension_name ] ) ) {
+							continue;
+						}
+
+						$dimension_value  = $dimension_filters[ $dimension_name ];
+						$dimension_filter = new Google_Service_AnalyticsReporting_DimensionFilter();
+						$dimension_filter->setDimensionName( $dimension_name );
+						$dimension_filter->setOperator( 'EXACT' );
+						$dimension_filter->setExpressions( array( $dimension_value ) );
+						$dimension_filter->setCaseSensitive( false );
```

Comment:
Let's remove this since the default is `false` and it isn't mentioned in the AC.
```suggestion
```